src/libudev/libudev.c

```diff
@@ -173,7 +173,9 @@ _public_ struct udev *udev_new(void) {
                                 if (prio < 0)
                                         log_debug("/etc/udev/udev.conf:%u: invalid log level '%s', ignoring.", line_nr, val);
                                 else
+#ifdef UDEV_PROPAGATE_LOG
                                         log_set_max_level(prio);
+#endif
```

Comment:
Is `udev_set_log_priority`  used at all ? I think we need to guard only  udev_new() is called from within udevd or one of its helpers.